understanding/20/timing-adjustable.html

```diff
@@ -34,59 +33,55 @@ <h2>Intent of Timing Adjustable</h2>
 
       <p>In some cases, however, it is not possible to change the time limit (for example, for an auction or other real-time event) and exceptions are therefore provided for those cases.</p>
-      
+
       <p>Content that operates on a timer does not need to be time adjustable if there is an alternative that does not rely on a timer. For example, a web application such as an email client provides notification of new email arriving with a temporary message (such as a 'toast' message) in the lower right-hand side of the interface, and the message disappears after 5 seconds. Users are able to identify the arrival of email through other means, such as viewing the Inbox, so the disappearance of the message does not set a time limit on the their ability to determine if new mail has arrived. If the user has no other means of discovering the same information (or performing the same function), then each message would need to meet this success criterion in order to provide users with sufficient time to access the information.</p>
 
-      <p>
-         						   
-         <strong>Notes regarding server time limits</strong>
-         					
-      </p>
-      
+      <h3>Notes regarding time limits</h3>
+
       <ul>
-         
          <li>Timed server redirects can be found below under Common Failures.</li>
-         
          <li>Non-timed server redirects (e.g., 3xx response codes) are not applicable because there
             is no time limit: they work instantly.
          </li>
-         
          <li>This success criterion applies only to time limits that are set by the content itself.
             For example, if a time limit is included in order to address security concerns, it
             would be considered to have been set by the content because it  is designed to be
             part of the presentation and interaction experience for that content. Time limits
             set externally to content, such as by the user agent or by factors intrinsic to the
             Internet are not under the author's control and not subject to WCAG conformance requirements.
             Time limits set by web servers should be under the author's/organization's control
-            and are covered. (Success Criteria 
-            <a href="no-timing" class="guideline">2.2.3</a>, 
-            <a href="interruptions" class="guideline">2.2.4</a> and 
-            <a href="re-authenticating" class="guideline">2.2.5</a> may also apply.)
+            and are covered. Success Criteria
+            <a href="no-timing" class="guideline">2.2.3 No Timing</a>,
+            <a href="interruptions" class="guideline">2.2.4 Interruptions</a> and
```

Comment:
nit: our style guide (via the Chicago Manual of Style) prefers serial commas

```suggestion
            <a href="interruptions" class="guideline">2.2.4 Interruptions</a>, and
```